### DISCUSSION TITLE
AntDeploy支持mac版本吗

### DISCUSSION THREAD

**Discussion by oldyangsir** [yuzd/AntDeploy#34] [Q&A] [answered]
如题

**Comment by yuzd** [comment, marked as answer]
antdeploy有一个mac版本是用xamarin.mac技术开发的
在qq群（488312978）的群文件有。
不过mac版本的功能相比winform 版本的功能要少一点，只支持iis，windows服务和 docker的发布。不支持linux的服务发布。
